src/iterative/KOSMAPOSL/KOSMAPOSLReconstruction.cxx

```diff
@@ -800,8 +823,12 @@ void KOSMAPOSLReconstruction<TargetT>::compute_kernelised_image(
                       kernelised_image_out[z][y][x] /= kernel_sum;
                     }
                   }
-                }
-}
+
+#ifdef STIR_OPENMP
+    }
+    #endif
+    }
+    }
```

Comment:
This is a bit hard to follow, Its just an extra brace for the OMP pragme?
Would be easier if you could make sure the closing indentation matches the opening.